sections/about/BoardSection.jsx

```diff
@@ -13,31 +15,72 @@ export const BoardSection = ({
       <div>
```

Comment:
Do you need wrapper (styled div) + inner unstyled div? (feel free to ignore if I'm wrong/it's needed)

Reply:
LOL no I don't good catch whoops